src/encoded/schemas/experiment.json

```diff
@@ -143,11 +143,19 @@
                         }
                     }
                 },
+                {
```

Comment:
You are putting dependency on experiment but upgrading all of the children of dataset? I don't think it will be enforced on anything but experiment.

Reply:
you are right
fixing